docs/guide/migrating-to-version-9.md

```diff
@@ -110,6 +122,31 @@ No changes.
 
 ### KInput
 
+#### Structure
+
+* `form-control` class is no longer present
+* `over-char-limit` class is no longer present
+* `has-error` class is no longer present
+* `help` class has been changed to `help-text`
+
+#### Constants, Types & Interfaces
+
+* `SizeArray` and `IconPositionArray` constants are no longer exported
+* `Size` and `IconPosition` types are no longer exported
+
+#### Props
+
+* `overlayLabel` prop is no longer supported
+* `size` prop is no longer supported
+* `iconPosition` prop is no longer supported
```

Comment:
"has been removed" instead of supported - I'd also state next to the `size` prop that all inputs are a single size